modules/hfs/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+if(NOT HAVE_CUDA)
+  ocv_module_disable(hfs)
```

Comment:
Does this module strongly depends on CUDA?

Reply:
yes, it can't run without cuda